Flutter/calendar/appointments.md

```diff
@@ -408,6 +408,47 @@ class _AppointmentDataSource extends CalendarDataSource {
 
 ![Adding recurrence appointment](images/appointments/recurrence-appointment.png)
 
+### Monthly recurring appointments on last day of every month
+
+For creating monthly recurring appointments on last day of every month by using BYMONTHDAY=-1in the flutter event calendar.
```

Comment:
Not monthly recurring appointment because it was used on yearly recurrence also

Reply:
I have checked and modified the content. I have updated in my latest commit